### PR TITLE
py-bottleneck: add 1.3.5 and py-versioneer: add 0.26

### PR DESCRIPTION
**Pull request by manuelakuhn**
switch `py-versioneer` to pypi

https://github.com/pydata/bottleneck/tree/v1.3.5
https://github.com/python-versioneer/python-versioneer/tree/0.26

### CODE REVIEW

**Review by scemama — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-versioneer/package.py [hunk 1]
```diff
@@ -10,12 +10,14 @@ class PyVersioneer(PythonPackage):
...
-    homepage = "https://github.com/warner/python-versioneer"
-    url = "https://github.com/warner/python-versioneer/archive/0.18.tar.gz"
-    git = "https://github.com/warner/python-versioneer.git"
+    homepage = "https://github.com/python-versioneer/python-versioneer"
+    pypi = "versioneer/versioneer-0.26.tar.gz"
+    git = "https://github.com/python-versioneer/python-versioneer.git"
 
     maintainers = ["scemama"]
 
+    version("0.26", sha256="84fc729aa296d1d26645a8f62f178019885ff6f9a1073b29a4a228270ac5257b")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-versioneer/package.py:19]: Confirmed sha256 for 0.26 BUT the URL change means the sha256 for v 0.18 is no longer valid.

Easiest thing to do would be to add the URL to v0.18.  Or you can change the sha256 for 0.18 to `ead1f78168150011189521b479d3a0dd2f55c94f5b07747b484fd693c3fbf335`

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Need to fix `py-versioneer` version for `0.18`.

File: var/spack/repos/builtin/packages/py-bottleneck/package.py [hunk 2]
```diff
@@ -9,13 +9,16 @@
 class PyBottleneck(PythonPackage):
     """A collection of fast NumPy array functions written in Cython."""
 
+    homepage = "https://github.com/pydata/bottleneck"
     pypi = "Bottleneck/Bottleneck-1.0.0.tar.gz"
 
+    version("1.3.5", sha256="2c0d27afe45351f6f421893362621804fa7dea14fe29a78eaa52d4323f646de7")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-bottleneck/package.py:15]: Confirmed the sha256.

File: var/spack/repos/builtin/packages/py-versioneer/package.py (hunk 1 above)
Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-versioneer/package.py:19]: Yes, you are right, I forgot about that. This is fixed now.
Thanks!

**Review by tldahlgren — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-versioneer/package.py [hunk 3]
```diff
@@ -10,12 +10,14 @@ class PyVersioneer(PythonPackage):
...
+    pypi = "versioneer/versioneer-0.26.tar.gz"
+    git = "https://github.com/python-versioneer/python-versioneer.git"
 
     maintainers = ["scemama"]
 
-    version("0.18", sha256="cf895b67f5bc62d61c4837458069ded8f66b4e5764c19f7253c51ee27e8b3a99")
+    version("0.26", sha256="84fc729aa296d1d26645a8f62f178019885ff6f9a1073b29a4a228270ac5257b")
+    version("0.18", sha256="ead1f78168150011189521b479d3a0dd2f55c94f5b07747b484fd693c3fbf335")
 
+    depends_on("python@3.6:", when="@0.19:", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-versioneer/package.py:23]: This should be 3.7:, at least in 0.26

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-versioneer/package.py (hunk 3 above)
Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-versioneer/package.py:23]: Done.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**